root/account/EditProfile.js

```diff
@@ -22,13 +21,17 @@ type Props = {
   +$c: CatalystContextT,
 };
 
-const EditProfile = ({$c, ...props}: Props) => {
+const EditProfile = ({
+  $c,
+  ...props
+}: Props): React.Element<typeof UserAccountLayout> | null => {
```

Comment:
`React.MixedElement` also works as a return type, as a generic way of saying it returns a React element without specifying which one. Though there is at least one case I saw where specifying it was useful.